### PR TITLE
feat: add private-key-converter to README.md

### PR DESCRIPTION
**Pull request by babblebey**
This PR adds the [babblebey/private-key-converter](https://github.com/babblebey/private-key-converter) to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,12 +212,18 @@ const { token, appId, expiration } = await githubAppJwt({
...
+Convert the format using `openssl` before passing it to your app.
 
 ```
 openssl pkcs8 -topk8 -inform PEM -outform PEM -nocrypt -in private-key.pem -out private-key-pkcs8.key
 ```
 
+- #### Using an Online Private Key Converter
+
+Convert quickly using the Web interface at https://private-key-converter.vercel.app
+
```
Comment (gr2m) [inline README.md:226]: it's the most convenient option, wanna put it on top?

Comment (babblebey) [reply README.md:226]: I just thought to be chill about it... I can surely lift it to the top 😃

Comment (gr2m) [reply README.md:226]: Make it so :rocket:

**Review by gr2m — APPROVED** [review]
Thanks

**Pull request merged**